src/cluster.h

```diff
@@ -86,15 +124,19 @@ typedef struct clusterLink {
  * kind of packet. PONG is the reply to ping, in the exact format as a PING,
  * while MEET is a special PING that forces the receiver to add the sender
  * as a node (if it is not already in the list). */
+/* redis 集群节点之前的通信的消息分为以下11种，
+ * 最后一种是包类型计数边界，代码中做判断
+ * 5（是否可以故障转移）,6（回复可以故障转移）,8（在手动故障转移的时候通知集群暂停处理客户端请求）三种包头没有包体
+ * 其他类型的包都由包头和包体两部分组成，包头格式相同，包体内容更具具体的类型填充 */
 #define CLUSTERMSG_TYPE_PING 0          /* Ping */
 #define CLUSTERMSG_TYPE_PONG 1          /* Pong (reply to Ping) */
 #define CLUSTERMSG_TYPE_MEET 2          /* Meet "let's join" message */
 #define CLUSTERMSG_TYPE_FAIL 3          /* Mark node xxx as failing */
 #define CLUSTERMSG_TYPE_PUBLISH 4       /* Pub/Sub Publish propagation */
-#define CLUSTERMSG_TYPE_FAILOVER_AUTH_REQUEST 5 /* May I failover? */
-#define CLUSTERMSG_TYPE_FAILOVER_AUTH_ACK 6     /* Yes, you have my vote */
+#define CLUSTERMSG_TYPE_FAILOVER_AUTH_REQUEST 5 /* May I failover? failover 授权请求包 */
```

Comment:
这里我们都新开行来做注释，我一般想着会少一点变更
```suggestion
/* failover 授权请求包 */
#define CLUSTERMSG_TYPE_FAILOVER_AUTH_REQUEST 5 /* May I failover? */
```